Makefile

```diff
@@ -186,6 +226,12 @@ src/linux-librem5:
 	@wget -c https://source.puri.sm/Librem5/linux-next/-/archive/pureos/5.9.16+librem5.2/linux-next-pureos-5.9.16+librem5.2.tar.gz
 	@tar -xvf linux-next-pureos-5.9.16+librem5.2.tar.gz --strip-components 1 -C src/linux-librem5
 
+src/linux-sdm845:
+	@echo "WGET linux-sdm845"
+	@mkdir src/linux-sdm845
+	@wget -c https://gitlab.com/sdm845-mainline/sdm845-linux/-/archive/sdm845-stable/sdm845-linux-sdm845-stable.tar.gz
```

Comment:
It might be better to fix this to a commit, otherwise as we update the kernel we could break stuff here...